training/replace_parameters.js

```diff
@@ -222,21 +222,32 @@ module.exports = class ParameterReplacer {
         if (ptype.isEntity)
             return this._getEntityListKey(ptype.type);
 
-        return [null, null];
+        if (!this._warned.has(pname + ':' + ptype)) {
+            console.log(`Found no values for ${pname}:${ptype}`);
+            this._warned.add(pname + ':' + ptype);
+        }
+
+        switch (pname) {
+        case 'message':
+        case 'snippet':
+        case 'text':
+        case 'description':
+        case 'status':
+        case 'body':
+            return ['string', 'tt:long_free_text'];
+
+        default:
+            return ['string', 'tt:short_free_text'];
```

Comment:
do we actually have `short_free_text`? at least it's not in the list https://almond.stanford.edu/thingpedia/strings